dali/test/python/test_operator_python_function.py

```diff
@@ -172,9 +180,56 @@ def test_python_operator_flip():
         for i in range(len(numpy_output)):
             assert numpy.array_equal(numpy_output.at(i), dali_output.at(i))
 
+
+class RotatePipeline(CommonPipeline):
+    def __init__(self, batch_size, num_threads, device_id, seed, image_dir):
+        super(RotatePipeline, self).__init__(batch_size, num_threads, device_id, seed, image_dir)
+        self.rotate=ops.Rotate(angle=90.0)
+    def define_graph(self):
+        images, labels = self.load()
+        rotate=self.rotate(images)
+        return rotate
+
```

Comment:
remove one empty line

Reply:
It follows the convention that every class or function is separated by two empty lines.